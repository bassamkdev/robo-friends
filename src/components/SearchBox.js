import React from 'react';

const SearchBox = ({searchChange}) => {
	return (
		<div className='pa3'>
			<input
			aria-label='search robots'
			className='pa3 ba b--green bg-lightest-blue'
			type='search'
			placeholder='search robot'
			onChange = {searchChange}
			/>
		</div>
	)
}
export default SearchBox;