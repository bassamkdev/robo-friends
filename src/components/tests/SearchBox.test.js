import React from 'react';

import SearchBox from '../SearchBox';

import { shallow } from 'enzyme';

it('render correctly SearchBox component', () => {
    const mockFunction = jest.fn();
    const wrapper = shallow(<SearchBox searchChange={mockFunction} />);
    expect(wrapper).toMatchSnapshot();
})