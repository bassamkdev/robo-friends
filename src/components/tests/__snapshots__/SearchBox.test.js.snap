// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly SearchBox component 1`] = `
<div
  className="pa3"
>
  <input
    aria-label="search robots"
    className="pa3 ba b--green bg-lightest-blue"
    onChange={[MockFunction]}
    placeholder="search robot"
    type="search"
  />
</div>
`;
